using-mobile-extensions/adobe-target/README.md

```diff
@@ -473,6 +474,11 @@ If you need to update SDK configuration, programmatically, use the following inf
 | target.clientcode | Client code for your account. |
 | target.timeout | Time, in seconds, to wait for a response from Target servers before timing out. |
 | target.environmentId | Environment ID you want to use, if this is left blank, the default production environment will be used. |
+| target.propertyToken | at_property token value which is generated from Target UI. If this is left blank, then no token will be sent in target network calls. |
```

Comment:
Please update the description so it reads as follows:

"at_property token value, which is generated from the Target UI. If this value is left blank, no token is sent in the Target network calls."

Reply:
done